src/HotChocolate/Filters/src/Types.Filters/FilterMiddlewareContext.cs

```diff
@@ -1,17 +1,19 @@
+using HotChocolate.Types.Filters.Conventions;
+
 namespace HotChocolate.Types.Filters
 {
     public class FilterMiddlewareContext
```

Comment:
what is the value of this? why not just use the convention in the middleware?

Reply:
because `DescirptorContext` is not available